src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1918,12 +1924,21 @@ namespace NServiceBus.Performance.TimeToBeReceived
         public System.TimeSpan MaxTime { get; }
     }
 }
-namespace NServiceBus.Persistence.Legacy
```

Comment:
:-1: for removing in this pull, how much work is this to split out? (I'm also not convinced to kill it off)